benchmarks/README.md

```diff
@@ -6,19 +6,19 @@ node http.js
 
 ## HTTP1
 
-Compare `apollo-datasource-http` (HTTP1 + Undici Pool) with apollo's `apollo-datasource-rest`  (HTTP1 + keepalive).
+Compare `apollo-datasource-http` (HTTP1 + Undici Pool) with apollo's `apollo-datasource-rest` (HTTP1 + keepalive).
 
 ```
 ❯ node benchmarks/http.js
 {
-  'apollo-datasource-rest (http1)': { startTime: 5974754539400n, endTime: 5975292928900n },
-  'apollo-datasource-http (http1)': { startTime: 5974751416200n, endTime: 5974986816000n }
+  'apollo-datasource-rest (http1)': { startTime: 12631206300610n, endTime: 12632239922219n },
+  'apollo-datasource-http (http1)': { startTime: 12631187804518n, endTime: 12631777587439n }
 }
-Results for 1000 subsequent requests: 
-apollo-datasource-rest (http1) | total time: 538389500ns (538.389ms)
-apollo-datasource-http (http1) | total time: 235399800ns (235.400ms)
+Results for 1000 subsequent requests:
+apollo-datasource-rest (http1) | total time: 1033621609ns (1033.622ms)
+apollo-datasource-http (http1) | total time: 589782921ns (589.783ms)
 ---
-apollo-datasource-http (http1) <> apollo-datasource-rest (http1) percent change: -56.277%
+apollo-datasource-http (http1) <> apollo-datasource-rest (http1) percent change: -42.940%
 ```
 
-**Result:** `apollo-datasource-http` is around `56%` faster than `apollo-datasource-rest`
+**Result:** `apollo-datasource-http` is around `43%` faster than `apollo-datasource-rest`
```

Comment:
This would mean a performance drop of more than 10%.

Reply:
I could verify it on my machine with even more (-15%). Can we improve this?

Reply:
I think it's mainly because of the overhead of caching the promises and I don't think it is a problem to be solved.

Reply:
Other possibilities:
1. The newer version of undici has lower performance. 
2. The node version is different than the last time you run the benchmark which improved the baseline.

Reply:
1. I verified the new undici version on main, and it has no performance drop.
2. No, I tested both with the same.

Please make your tests.